example/tests/unit/test_default_drf_serializers.py

```diff
@@ -73,3 +76,24 @@ def test_render_format_keys(settings):
 
     result = json.loads(rendered.decode())
     assert result['data']['attributes']['json-field'] == {'json-key': 'JsonValue'}
+
+
+class TestDRFBlogViewSet(TestBlogViewSet):
```

Comment:
as all other tests are written in pytest in this module I would write these tests the same way. This way we do not have a dependency on TestBlogViewSet (which basically only creates an entry during setup phase anyway).